docs/en/connector-v2/sink/Kafka.md

```diff
@@ -50,6 +51,23 @@ In AT_LEAST_ONCE, producer will wait for all outstanding messages in the Kafka b
 
 NON does not provide any guarantees: messages may be lost in case of issues on the Kafka broker and messages may be duplicated.
 
+### partition_key [string]
+
+Determine the partition of the kafka send message based on the key.
```

Comment:
Replace with `Configure which field is used as the key of the kafka message` better.